### ISSUE TITLE
ios cannot compile

### ISSUE THREAD

**Issue by crly** [aarongrider/vision-camera-ocr#30]
xcode15
react-native-vision-camera : 3.1.0
vision-camera-ocr: 1.0

This library needs to be updated

**Comment by MariuzM** [comment]
Yep same for me getting `CommandError: Failed to build iOS project. "xcodebuild" exited with error code 65.`

**Comment by MariuzM** [comment]
Actually i started getting this when i installed `vision-camera-ocr` before that was working

**Comment by MariuzM** [comment]
Looks like there is PR for this https://github.com/aarongrider/vision-camera-ocr/pull/32
